.gitpod/Dockerfile

```diff
@@ -20,7 +20,7 @@ SHELL ["/bin/bash" , "-c"]
 # Set up buildpack
 COPY --from=buildpack /usr/local/bin/ /usr/local/bin/
 COPY --from=buildpack /usr/local/buildpack/ /usr/local/buildpack/
-RUN install-buildpack
+RUN install-buildpack && usermod -g ${USER_ID} ${USER_NAME}
```

Comment:
so the primary group needs to be gitpod?

Reply:
No, only the GID needs to be 33333, for some reason it wasn't being set to that as intended.

Reply:
ok, the primary gid was set to `0`, which was intentional to allow write access to specific folders and support Openstack.

Reply:
> ok, the primary gid was set to 0, which was intentional to allow write access to specific folders and support Openstack.

If I understand your setup correctly, `DOCKERD_ARGS` might be helpful. You can refer to [this link](https://www.gitpod.io/docs/configure/projects/environment-variables#user-specific-environment-variables) for more information.

> So the primary group (gid) needs to be 33333 to work?

I think so.